### PR TITLE
Update Update-SPODataEncryptionPolicy.md

### PR DESCRIPTION
**Pull request by sathana87**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

File: sharepoint/sharepoint-ps/sharepoint-online/Update-SPODataEncryptionPolicy.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Update-SPODataEncryptionPolicy [-KeyVaultName] <String> [-KeyName] <String> [-Ke
 
 ## DESCRIPTION
 
-Use the Update-DataEncryptionPolicy cmdlet to update customer encryption status for your geo tenant.
+Use the Update -SPODataEncryptionPolicy cmdlet to update customer encryption status for your geo tenant.
```
Comment (dariomws) [inline sharepoint/sharepoint-ps/sharepoint-online/Update-SPODataEncryptionPolicy.md:27]:
```suggestion
Use the Update-SPODataEncryptionPolicy cmdlet to update customer encryption status for your geo tenant.
```

**Review by dariomws — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]
Copy review - approved with @dariomw's suggested change implemented.

**Review by tseward — APPROVED** [review]
Also approved with @dariomw's changes.

**Comment by yogkumgit** [comment]
@get-itips

**Review by get-itips — CHANGES_REQUESTED** [review]
I am ok once Dario's suggestion is applied

**Comment by yogkumgit** [comment]
@sathana87 Please implement the suggested changes.

**Comment by yogkumgit** [comment]
@sathana87 Gentle reminder.

**Comment by yogkumgit** [comment]
@tiburd Please help.

**Pull request merged**